### PR TITLE
✨ Add pubsub module

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/server/api/routes/nft/mint-subscription.js [hunk 1]
```diff
@@ -43,6 +44,12 @@ router.post('/nft/mint-subscription', async (req, res, next) => {
       });
       return { subscriptionId };
     });
+    publisher.publish(PUBSUB_TOPIC_MISC, req, {
+      logType: 'subscribeMintNFTRequest',
```
Comment (nwingt) [inline src/server/api/routes/nft/mint-subscription.js:48]:
```suggestion
      logType: 'UserCreatorFollowUnverified',
```

File: src/server/api/routes/nft/mint-subscription.js [hunk 2]
```diff
@@ -96,6 +103,12 @@ router.put('/nft/mint-subscription/:id', async (req, res, next) => {
       }
       await t.update(docRef, { isVerified: true });
     });
+    publisher.publish(PUBSUB_TOPIC_MISC, req, {
+      logType: 'SubscribeMintNFTVerified',
```
Comment (nwingt) [inline src/server/api/routes/nft/mint-subscription.js:107]:
```suggestion
      logType: 'UserCreatorFollow',
```

File: src/server/api/routes/nft/mint-subscription.js [hunk 3]
```diff
@@ -110,6 +123,12 @@ router.delete('/nft/mint-subscription/:id', async (req, res, next) => {
   const { id: subscriptionId } = req.params;
   try {
     await nftMintSubscriptionCollection.doc(subscriptionId).delete();
+    publisher.publish(PUBSUB_TOPIC_MISC, req, {
+      logType: 'SubscribeMintNFTRemoved',
```
Comment (nwingt) [inline src/server/api/routes/nft/mint-subscription.js:127]:
```suggestion
      logType: 'UserCreatorUnfollow,
```

**Review by nwingt — COMMENTED** [review]
Suggest renaming `logType` for reusing in `users/follow` later

Also, consider renaming:
`subscriberEmail` -> `followerEmail`
`subscribedWallet` -> `followeeWallet`

**Comment by williamchong** [comment]
how about just email and creator wallet?

**Review by nwingt — APPROVED** [review]

**Pull request merged**